data/data/bootstrap/files/usr/local/bin/bootkube.sh.template

```diff
@@ -294,6 +295,30 @@ then
     record_service_stage_success
 fi
 
+if [ ! -f node-tuning-bootstrap.done ]
+then
+    record_service_stage_start "node-tuning-bootstrap"
+	echo "Rendering Node Tuning core manifests..."
+
+	rm --recursive --force node-tuning-bootstrap
+
+	bootkube_podman_run \
+		--name node-tuning-render \
+		--volume "$PWD:/assets:z" \
+		"${NODE_TUNING_OPERATOR_IMAGE}" \
+		render \
+		--asset-input-dir=/assets/manifests \
```

Comment:
is `assets/manifests` the default input dir ?

Reply:
Yup, one of the first things this script does is copy over `/openshift` assets to the `/manifests` folder, so that will contain all the relevant files for us.

https://github.com/openshift/installer/pull/6691/files#diff-f6cfc5d463e8139ffa14ab1b1b448280664b32bfb20b60cef2479e27561d6a33R76